README.md

```diff
@@ -1,5 +1,44 @@
-# mlops-kube-gateway
+# Neu.ro MLFlow2Seldon deployer
 
-Web service for integration of third-party tools that require direct access to k8s (like Seldon, Algorithmia, MLFlow) with Neu.ro platform.
+An integration service to deploy [MLFlow registered model](https://www.mlflow.org/docs/latest/model-registry.html) as REST/GRPC API to Kubernetes cluster using Seldon-core.
 
-Maintained by Neu.ro MLOps team.
+# Usage
+This service is running inside of the Kubernetes cluster, where the Seldon-core is deployed.
+By constantly fetching the MLFlow server registered models (running as a platform job) via MLFlow Python SDK, it synchronizes the MLFlow state to Seldon-core within the Kubernetes cluster.
+
+For instance, if the MLFlow registered model version gets assigned to the Staging/Production stage, the corresponding model binary gets deployed from the MLFlow into the K8s cluster as the SeldonDeployment (exposing REST/GRPC APIs).
+If the stage assignment gets removed/updated - the corresponding SeldonDeployment is changed respectively.
+
+Given that, all the interaction with the service is done implicitly via the MLFlow server state. There is no need to execute particular commands/workloads against this service directly.
+
+## Prerequisites and usage assumptions
+- MLFlow 
+    - is up and running as a [platform job](https://github.com/neuro-actions/mlflow)
+    - disabled platform SSO;
```

Comment:
u mean --no-http-auth?

Reply:
aaaahhh it's running inside a jooooobbb

Reply:
ok that makes sense -- I agree, mlflow inside a job works as fine as a deployed version, for demo purposes

Reply:
hmm, yet it's weird to make a k8s-native app (hell gates) to be dependant on a neuro job.. but I don't have strong arguments for that

Reply:
I do understand your thoughts)
but, we have what we have 😄 , hope potential SAML authentication will help us)